app/src/main/java/com/android/partagix/ui/components/ItemList.kt

```diff
@@ -99,11 +100,21 @@ fun ItemList(
                   loan[index]
                 }
               },
+          available =
+              if (availability.isEmpty()) {
+                true
+              } else {
+                if (availability.size <= index) {
+                  true
+                } else {
+                  availability[index]
+                }
+              },
```

Comment:
For better readability & less cognitive complexity:

```suggestion
          available = availability.getOrElse(index) { true },
```